daemon/cmd/local_node_sync_test.go

```diff
@@ -15,6 +15,8 @@ import (
 	k8sRuntime "k8s.io/apimachinery/pkg/runtime"
 	k8stypes "k8s.io/apimachinery/pkg/types"
 
+	"github.com/cilium/stream"
+
```

Comment:
It would be probably great to have a [linter](https://github.com/daixiang0/gci) enforcing that the `github.com/cilium/*` imports form a separate block, otherwise they'll soon start to get coalescenced with the ones above.

Reply:
Hm. I would've expected `goimports` to flag this, but I'm not sure how we configured it. In any case, do we actually want a third block?

Reply:
AFAIR `goimports` is more opinionated and cannot be configured (except for the local-prefixes which form the third block) -- I had tested it, and it would not flag this in either way -- while `gci` allows you to [configure the specific format for the imports](https://golangci-lint.run/usage/linters/#gci).

> In any case, do we actually want a third block?

I'm not sure honestly, although generally speaking it seems reasonable to me. Regardless, it is probably a good idea to pick a format and strictly enforce it, to remove this ambiguity.